<ui version="4.0" >
 <class>Solidproc</class>
 <widget class="QWidget" name="Solidproc" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>399</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QListWidget" name="listWidgetProcessors" >
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupProcInfo" >
     <property name="title" >
      <string>Selected Processor</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="labelVendor" >
        <property name="text" >
         <string>Vendor</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>lineVendor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="KLineEdit" name="lineVendor" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="labelModel" >
        <property name="text" >
         <string>Model:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>lineModel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="KLineEdit" name="lineModel" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="labelSpeed" >
        <property name="text" >
         <string>Speed:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>lineSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="KLineEdit" name="lineSpeed" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLabel" name="labelExtensions" >
        <property name="text" >
         <string>Supported extensions:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>lineExtensions</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="KLineEdit" name="lineExtensions" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
